### DISCUSSION TITLE
currentSchema parameter is ignored

### DISCUSSION THREAD

**Discussion by jstec** [pgbouncer/pgbouncer#572] [Q&A]
pgBouncer is ignoring currentSchema parameter in my database connection string. 
How can I fix that?

Regards

**Comment by petere** [comment]
I don't know what `currentSchema` is or what it's supposed to do, so ignoring it is probably not wrong.

**Reply by arbansal01** [reply]
@jstec do you mean pointing pgbouncer to current schema?
for that try this query
SET search_path TO myschema;

**Comment by mbanck** [comment]
It's a pgJDBC connection parameter, see https://jdbc.postgresql.org/documentation/head/connect.html

I got some people trying to connect to Postgres from SAP Emarsys (JDBC-based) via pgBouncer, and they need to set the schema. If they do so via `currentSchema=foo`, I see (apparently pgJDBC rewrites this?)
```
2022-02-16 15:25:43.519 UTC [8906] LOG C-0xe0aed0: (nodb)/(nouser)@XXX.XXX.XXX.XXX:20354 closing because: unsupported startup parameter: search_path (age=0s)
```
